xml/System.Text.Json/Utf8JsonWriter.xml

```diff
@@ -277,10 +342,12 @@
         <Parameter Name="escape" Type="System.Boolean" />
       </Parameters>
       <Docs>
-        <param name="utf8Value">To be added.</param>
-        <param name="escape">To be added.</param>
-        <summary>To be added.</summary>
+        <param name="utf8Value">The UTF-8 encoded value to be written as a JSON comment.</param>
+        <param name="escape"><see langword="false" /> to indicate the writer should assume the property name is properly escaped and skips the escaping step; otherwise, <see langword="true" />. This is an optional parameter and its default value is <see langword="true" />.</param>
```

Comment:
This API doesn't contain "property name". Looks like copy/paste error.

writer should assume the ~property name~ value is properly escaped

```suggestion
        <param name="escape"><see langword="false" /> to indicate the writer should assume the value is properly escaped and skips the escaping step; otherwise, <see langword="true" />. This is an optional parameter and its default value is <see langword="true" />.</param>
```